tests/munging/test_str.py

```diff
@@ -126,3 +126,14 @@ def test_split_into_nhot_long(seed, st):
     assert set(f1.names) == set(fr.names)
     f1 = f1[..., fr.names]
     assert f1.to_list() == fr.to_list()
+
+
+def test_split_into_nhot_view():
+  f0 = dt.Frame(A=["cat,dog,mouse", "mouse", None, "dog, cat"])
+  f1 = dt.split_into_nhot(f0[::-1, :])
```

Comment:
Does `f0[::-1, :]` mean all the rows taken in a reverse order?

Reply:
yes, same rules as with a regular python array